### PR TITLE
Bug fixes: logout

### LINKED ISSUE #1810: Logout button not working
### My actions before raising this issue
- [ X ] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [ X ] Searched [past issues](/issues)

### Expected Behaviour
When clicking on the Logout but, get the user session closed.

### Current Behaviour
When clicking on the Logout but, nothing is happening.

### Steps to Reproduce (for bugs)
Follow the default installation process.

### Your Environment
- Git hash commit (`git log -1`):
commit db29291d43172aa33194a6fb28f8000309d38344 (HEAD -> develop, origin/develop, origin/HEAD)
Author: Andrey Zhavoronkov <41117609+azhavoro@users.noreply.github.com>
Date:   Thu Jun 25 22:28:01 2020 +0300

    added hash to js bundle names (#1803)

- Docker version `docker version` (e.g. Docker 17.0.05):
Client: Docker Engine - Community
 Version:           19.03.12
 API version:       1.40
 Go version:        go1.13.10
 Git commit:        48a66213fe
 Built:             Mon Jun 22 15:45:36 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.12
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.10
  Git commit:       48a66213fe
  Built:            Mon Jun 22 15:44:07 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683

- Are you using Docker Swarm or Kubernetes?
No

- Operating System and version (e.g. Linux, Windows, MacOS):
Ubuntu 18.04.04

- Other diagnostic information / logs:
  No errors in the cvat logs
  Server version: 1.1
  Core version: 3.1.0
  Canvas version: 1.2.1
  UI version: 1.4.2

### PR DESCRIPTION
**Pull request by lizhming**
### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

Resolve #1810

### CODE REVIEW

**Comment by ActiveChooN** [comment]
@lizhming, thanks a lot, could you just add changes to changelog?

**Comment by bsekachev** [comment]
@lizhming 

Hi, thanks for the PR. Please, fill out checkboxes in PR description. We aren't permitted to merge it without some of them.

File: CHANGELOG.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - A couple of exceptions in AAM related with early object activation (<https://github.com/opencv/cvat/pull/1755>)
 - Propagation from the latest frame (<https://github.com/opencv/cvat/pull/1800>)
 - Number attribute value validation (didn't work well with floats) (<https://github.com/opencv/cvat/pull/1800>)
-
+- Fixed Logout function  (<https://github.com/opencv/cvat/issues/1810>)
```
Comment (bsekachev) [inline CHANGELOG.md:57]: This is place for the PR link. Not for the issue.

**Comment by nmanovic** [comment]
@lizhming , thanks for the PR. Great to see such contributions.

**Pull request merged**